README.md

```diff
@@ -81,38 +81,55 @@ structure.
 
 ## Development
 
-To make changes to the playbook theme, you will need to install the
-dependencies:
+To make changes to the playbook theme, you must have the following installed
+on your local machine:
+
+* Ruby >= 2.6
+* Node.js >= 12
+* Go >= 1.13
+
+Assuming this is sorted, you can install the dependencies:
 
 ```
-npm install
-go get github.com/errata-ai/vale
+script/bootstrap
 ```
 
 Then to start a local development server run:
 
 ```
-npm start
+script/server
 ```
 
-You can now visit [localhost:3000](http://localhost:3000) to see your local
-playbook. Theme files live in the `/build` directory
+You can now visit [localhost:4000](http://localhost:4000) to see your local
+playbook. CSS files live in the `build/css` directory, while Javascript
```

Comment:
Can we standardize, so css goes in `_stylesheets` or something?

Reply:
Yep, good shout. I've moved it all over to Sass too, so we can build it all with Webpack